src/ServiceInsight/Framework/Commands/ObservableCommand.cs

```diff
@@ -0,0 +1,62 @@
+namespace ServiceInsight.Framework.Commands
+{
+    using System;
+    using System.Reactive.Disposables;
+    using System.Reactive.Linq;
+    using System.Threading;
+    using System.Windows.Input;
+
+    class ObservableCommand : ICommand
```

Comment:
Does this class need to be thread-safe WRT accessing the `isExecuting` and `latest` fields? (I noticed the use `Interlocked.Exchange` in `Dispose`.)